tranzito_utils_js/dist/period_selector.js

```diff
@@ -16,100 +16,90 @@ class PeriodSelector {
     return urlParams
   }
 
-  urlParamsWithNewPeriod(selectedPeriod = null) {
+  urlParamsWithNewPeriod (selectedPeriod = null) {
     // If it's null, empty, undefined or anything other than string
     if (typeof selectedPeriod !== 'string') {
       // If not passed the period, Figure it out, defaulting to all
-      selectedPeriod =
-        $('#timeSelectionBtnGroup .btn.active').attr('data-period') || 'all'
+      selectedPeriod = document.querySelector('#timeSelectionBtnGroup .btn.active').attributes['data-period']?.value || 'all'
     }
 
     const urlParams = this.urlParamsWithoutPeriod()
     urlParams.append('period', selectedPeriod)
 
     if (selectedPeriod === 'custom') {
-      urlParams.append('start_time', $('#start_time_selector').val())
-      urlParams.append('end_time', $('#end_time_selector').val())
+      urlParams.append('start_time', document.getElementById('start_time_selector').value)
+      urlParams.append('end_time', document.getElementById('end_time_selector').value)
     }
     return urlParams
   }
 
-  enableCustomSelection() {
+  enableCustomSelection () {
     // Show the custom selection inputs
-    $('#timeSelectionBtnGroup .period-select-standard').removeClass('active')
-    $('#periodSelectCustom').addClass('active')
-    $('#timeSelectionBtnGroup').addClass('custom-period-selected')
-    // Sometimes bootstrap isn't loaded
-    if (typeof $().collapse == 'function') {
```

Comment:
Can we keep the check for bootstrap? If it is loaded, it would be nice to use it rather than manual hiding.

Reply:
I think its not possible without having jQuery installed.

Reply:
You could do `typeof $()?.collapse`